CHANGELOG.md

```
@@ -40,7 +40,7 @@ This changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.
 - docs: create workflow best practices guide (`docs/workflows/03-workflow-best-practices.md`) (`#1743`)
 - Fixed broken `MAINTAINERS.md` relative link in `docs/sdk_developers/bug.md` by using the repository-root GitHub URL. (#1666)
 - docs(setup): specify unit tests for local setup verification. (#1856)
-- doc: Fix testnet link in README.md. (#1879)
+- doc: Fix testnet link in README.md. (#1879) 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)